core/scss/components/data-table/data-table.scss

```diff
@@ -4,6 +4,22 @@
   .md-data-table {
     position: relative;
 
+    &.zebra {
```

Comment:
Wouldn't this be md-data-table--zebra? A standalone would make sense if we were using it in other places.